meson.build

```diff
@@ -1825,6 +1825,21 @@ if get_option('cplayer')
             command: [osxbundle, mpv.full_path(), '@SOURCE_ROOT@'],
         )
     endif
+
+    if meson.can_run_host_binaries()
+        mpv_desktop_path = join_paths(source_root, 'etc', 'mpv.desktop')
+        mpv_bin = win32 ? mpv_com : mpv
+        mpv_exes = win32 ? [mpv, mpv_com] : mpv
```

Comment:
shouldn't we just skip the desktop file for win32?

Reply:
We could. Should we skip it for macOS too? Only linux/bsd uses it right?

Reply:
yea